extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/cache/OffHeapNamespaceExtractionCacheManager.java

```diff
@@ -56,15 +53,11 @@
   @Inject
   public OffHeapNamespaceExtractionCacheManager(
       Lifecycle lifecycle,
-      @Named("namespaceExtractionFunctionCache")
-      ConcurrentMap<String, Function<String, String>> fnCache,
-      @Named("namespaceReverseExtractionFunctionCache")
-      ConcurrentMap<String, Function<String, List<String>>> reverseFnCache,
       ServiceEmitter emitter,
-      final Map<Class<? extends ExtractionNamespace>, ExtractionNamespaceFunctionFactory<?>> namespaceFunctionFactoryMap
+      final Map<Class<? extends ExtractionNamespace>, ExtractionNamespaceCacheFactory<?>> namespaceFunctionFactoryMap
```

Comment:
i am not sure if this impl needs more documentation like the size of the cache ?